packages/example/src/data/nav-items.yaml

```diff
@@ -4,6 +4,12 @@
 - title: Demo
   pages:
     - path: /demo
+- title: Gallery
+  pages:
+    - path: /gallery
+- title: Contributing
```

Comment:
```suggestion
- title: Contributions
```

Reply:
You'll probably have to rename the file, too. This is what we're calling it on the Carbon website, should keep it consistent.